docs/querying/api.md

```diff
@@ -699,6 +701,41 @@ curl -G http://localhost:9090/api/v1/metadata?limit=2
 }
 ```
 
+### Finding metadata for a single metric
+
+It returns metadata about the provided metric name currently scrapped from targets. However, it does not provide any target information.
+This is considered **experimental** and might change in the future.
+
+```
+GET /api/v1/metadata/<metric_name>
```

Comment:
We usually use URL parameters

Reply:
I noticed that, but the main reason for it is to circumvent URL character limits which I feel don't apply to our situation which is very similar to [querying label values](https://prometheus.io/docs/prometheus/latest/querying/api/#querying-label-values) (as these don't also have a limit).

I'm happy to change it if you think otherwise.

Reply:
This doesn't help you get around URL length limits as it's still in the URL, we add POST where that's a concern.

The label values appears to be inconsistent for no good reason.